bokeh/core/_templates/autoload_js.js

```diff
@@ -91,6 +91,8 @@ calls it with the rendered model.
       document.body.appendChild(element);
     }
 
+    const hashes = {{ bundle.hashes|json }};
```

Comment:
I think this PR needs a rebase on current `branch-2.4`? These changes are restoring codepaths that were intentionally removed in a previous PR.

Reply:
Whenever I pulled after rebasing these 2 files would update. I tried rebasing and then force pushing now

Reply:
❯ git remote get-url upstream
https://github.com/bokeh/bokeh.git

❯ git rebase --onto upstream/branch-2.4 HEAD~1
`

Reply:
@IuryPiva did you fetch the most recent `branch-2.4` before the rebase? If not, you need to do that first. I would actually expect a force push is necessary after the rebase.

Reply:
I did, and it worked on [11060](https://github.com/bokeh/bokeh/pull/11060). I will try again

Reply:
I'm not sure what's going on, it actually added a third copy of that block. @tcmetzger is going to add you to @bokeh/dev so that you can work off a clone of the main repo instead of a fork. I think it might be simplest just to submit new PRs after that is done.

Reply:
@Iurypiva you should have an invite to @bokeh/dev now